src/chrome/content/rules/National_Geographic.com.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+
+	Redirect to HTTP:
+
+		- adventureblog
+		- channel
+		- help
+		- yourshot
+		- yourshotblog
+	
+	Mixed content:
+
+		- archive
+		- intelligenttravel		
+		- news
+		- press
+		- video
+
+-->
+<ruleset name="National Geographic">
+
+	<target host="www.nationalgeographic.com" />
```

Comment:
Please also add

```xml
	<target host="nationalgeographic.com" />

	<rule from="^http://nationalgeographic.com/"
		to="https://www.nationalgeographic.com/" />
```

with a note in the top comment explaining that `^` times out over `https`.